lib/trino-parquet/pom.xml

```diff
@@ -178,4 +178,29 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <version>3.0.0</version>
+                <executions>
+                    <execution>
+                        <id>generate-build-id</id>
+                        <phase>generate-resources</phase>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <executable>${project.basedir}/bin/generate_build_id.sh</executable>
+                    <arguments>
+                        <argument>${project.basedir}/src/main/resources/io/trino/parquet/writer/build.properties</argument>
```

Comment:
The gitcommitid plugin already writes the commit id (and build time) in the manifest - see https://github.com/airlift/airbase/blob/831369f2be2c8bfe4b585e30a0082bbf01a2ef68/airbase/pom.xml#L548-L553.

Reply:
See example at https://github.com/trinodb/trino/blame/1523f283f52fcba45497b48124dd3f86f725ea50/client/trino-jdbc/src/main/java/io/trino/jdbc/TrinoDriver.java#L46 for how this can be accessed.

Reply:
Anyway, this isn't gonna be needed.

Reply:
Okay, I see that now. @findepi are you suggesting to break convention with other parquet writers and use our implementation version or to read in the manifest and parse its attributes to build the createdBy field?

Reply:
I think https://github.com/trinodb/trino/pull/9157#discussion_r716483129 answers it?